### ISSUE TITLE
Suggestion: One Audio Track at a time

### ISSUE THREAD

**Issue by JoyBell2** [stax76/staxrip#411]
Stax currently encodes all the audio tracks two at a time and then when done starts the video encoding.
I would suggest flipping the order:
Always start the video encoding and then up to one audio track at a time as well.
The audio tracks do not take much processing but can often thrash the hard drives for a long time, this is exasperated when two audio tracks are thrashing the disk at the same time and storage speeds slow to a crawl.
I have had encodes spend an hour on the audio tracks before starting on the video encode.

**Comment by Revan654** [comment]
If Audio encoding is bringing your PC to a crawl then there some other underlining issues with your PC. a single Harddrive (7200rpm) can write at 80MBs+. SSD can write even faster. Which is plenty of some simple audio.

You can alter the Parallel process in the Settings. The Default is set to 2.

**Comment by JoyBell2** [comment]
The uncompressed raw WAV files that the encoder makes as intermediary states which are all over 5GB each that's the slow down.
I am not running sub standard equipment.

**Comment by Revan654** [comment]
Whatever the case may be, Stax76 is retired and my forked version I don't plan on changing how the audio is processed.

**Comment by JoyBell2** [comment]
Crap, I thought I was in the Forked Version 1.8.2.0. This is where the crash dialog took me. Very sorry. 
Also I can't find the Forked Version page.

**Comment by Revan654** [comment]
It takes awhile to update everything inside the software. Updating the URL had a low priority during the first release. New URL will be included with the next release.

Fork: https://github.com/Revan654/staxrip
